src/components/fields/MultiTagArrayField.js

```diff
@@ -0,0 +1,163 @@
+import * as React from "react";
+import * as PropTypes from "prop-types";
+import BaseComponent from "../BaseComponent";
+import {classNames, getUiOptions} from "../../utils";
+import ReactContext from "../../ReactContext";
+import {Affix, Button} from "../components";
+
+
+@BaseComponent
+export default class MultiTagArrayField extends React.Component {
```

Comment:
I highly recommend writing new components in TypeScript. `SortArrayField.tsx` could be a good reference for best practices of how to write LajiForm components in 2023.